File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.rs
use std::cmp;
impl Solution {
    pub fn max_profit(prices: Vec<i32>) -> i32 {
        let mut min = i32::MAX;
        let mut ans = 0;

        for i in prices{
            min = cmp::min(min,i);
            ans = cmp::max(ans, i - min);
        }
        ans
    }
}

File: Data Structures & Algorithms/number-of-senior-citizens/submission-0.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        char s1,s2;
        int ans = 0;
        for (auto i : details){
            s1 = i[11];
            s2 = i[12];
            int a,b;
            a = s1-'0';
            b = s2 - '0';
            int num = a*10 + b;

            if (num > 60){
                ans++;
            }
        }
        return ans;
    }
};